src/components/Navbar/NavBar.tsx

```diff
@@ -296,10 +318,65 @@ const ReactNavLink = styled(Link)`
     color: black;
   }
   :hover {
-    color: #9e9e9e;
-  }
+    color: #9e9e9e;}
+  position:relative;
 `;
 
+
 const Close = styled(CloseIcon)`
   cursor: pointer;
+`;
+
+
+const Dropdown = styled.div`
+  display:flex;
+  z-index: 11;
+  display: block;
+  position: absolute;
```

Comment:
There is a lot going on here. Why do you need display block, flex and position absolute?